src/main/org/deidentifier/arx/distributed/ARXPartition.java

```diff
@@ -36,8 +31,44 @@
  */
 public class ARXPartition {
 
+    /** Data*/
+    private final DataHandle data;
+    /** Subset*/
+    private final int[] subset;
+
+    /**
+     * Creates a new instance
+     * @param handle
+     * @param subset
+     */
+    public ARXPartition(DataHandle handle, int[] subset) {
+        this.data = handle;
+        this.subset = subset;
+    }
+
+    /**
+     * Returns the data
+     * @return
+     */
+    public DataHandle getData() {
+        return data;
+    }
+
+    /**
+     * Returns the subset, if any
+     * @return
+     */
+    public int[] getSubset() {
+        return subset;
+    }
+
     /** Random */
-    private static final Random random = new Random(0xDEADBEEF);
+    private static Random random = new Random();
+
+    public static void setRandomSeed(long seed) {
```

Comment:
I dont like that this is a method that turns this into deterministic by replacing the RNG. Ideally, this should be set through the constructor, so that random can stay final.